README.md

```diff
@@ -176,7 +176,8 @@ To use SwiftGen, simply create a `swiftgen.yml` YAML file to list all the subcom
 
 ```yaml
 strings:
-  inputs: Resources/Base.lproj/Localizable.strings
+  inputs: Resources/Base.lproj
+  filter: Localizable.*\.strings
```

Comment:
This filter example is quite confusing. Either you want to select all the `.strings` files, so use `.*\.strings`, or you want to select all `Localizable.strings` files, so use `Localizable\.strings`… But it's quite uncommon to have `LocalizableFoo.strings` and `LocalizableBar.string` and `Localizable.*\.strings` here, so maybe not the best example here. Maybe use the same example as you did in `ConfigFile.md`?